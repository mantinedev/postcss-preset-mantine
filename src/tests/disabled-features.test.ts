import { testTransform } from './utils';

const rem = `
.card {
  padding: rem(32px);
}
`;

const lightDark = `
.card {
  color: light-dark(#000, #fff);
}
`;

describe('disabled-features', () => {
  it('does not transform rem if rem function is disabled', async () => {
    const res = await testTransform(rem, {
      features: { remEmFunctions: false },
    });
    expect(res.css).toMatchSnapshot();
  });

  it('does not transform light/dark if lightDarkFunction is disabled', async () => {
    const res = await testTransform(lightDark, {
      features: { lightDarkFunction: false },
    });
    expect(res.css).toMatchSnapshot();
  });
});
