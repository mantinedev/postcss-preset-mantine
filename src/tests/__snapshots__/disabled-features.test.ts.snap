// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`disabled-features does not transform light/dark if lightDarkFunction is disabled 1`] = `
"
.card {
  color: light-dark(#000, #fff);
}
"
`;

exports[`disabled-features does not transform rem if rem function is disabled 1`] = `
"
.card {
  padding: rem(32px);
}
"
`;
